README.md

```diff
@@ -1,24 +1,73 @@
-# Delegated Credentials for TLS
 
-This is the working area for the IETF [TLS Working Group](https://datatracker.ietf.org/wg/tls/documents/) Internet-Draft, "Delegated Credentials for TLS".
 
-* [Editor's Copy](https://tlswg.github.io/tls-subcerts/#go.draft-ietf-tls-subcerts.html)
-* [Working Group Draft](https://tools.ietf.org/html/draft-ietf-tls-subcerts)
-* [Compare Editor's Copy to Working Group Draft](https://tlswg.github.io/tls-subcerts/#go.draft-ietf-tls-subcerts.diff)
 
-## Building the Draft
+# AUTH48 for RFC-to-be 9345 <draft-ietf-tls-subcerts-15>
```

Comment:
Rather than putting your content into `README.md`, this content should go in the comments of your pull request, rather than checking it into the repo itself.  (The comment field also accepts Markdown.)  You could make the title of the PR "AUTH48 for RFC-to-be 9345", and put the content here in the comments.

If you want to have a file in the repo for some reason (e.g., so that it lives longer than one PR), please make it something separate, say `AUTH48.md`.  Then we can delete it when AUTH48 is over.